CMakeLists.txt

```diff
@@ -19,15 +19,66 @@ endif()
 find_package(NumPy REQUIRED)
 find_package(PythonExtensions REQUIRED)
 find_package(Python COMPONENTS Interpreter Development)
+include(FetchContent)
 
 set(RF_BASE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
-set(TF_BUILD_CUDA OFF CACHE BOOL "Enables build of CUDA code")
-set(TF_BUILD_TESTS OFF CACHE BOOL "Enables build of tests")
-set(TF_BUILD_EXAMPLES OFF CACHE BOOL "Enables build of examples")
-add_subdirectory(extern/taskflow EXCLUDE_FROM_ALL)
-add_subdirectory(extern/rapidfuzz-cpp)
-add_subdirectory(extern/jarowinkler-cpp)
+# todo this appears to be broken somehow
+# results in linker errors
+find_package(Taskflow 2.7.0 EXACT QUIET)
```

Comment:
Why `EXACT`?

Reply:
This is discussed here: https://github.com/maxbachmann/Levenshtein/issues/22. It was done for C++14 support. Since anyone who has a newer version of Taskflow installed should have a C++17 compatible compiler, I decided to relax these requirements. This was already done on main: https://github.com/maxbachmann/RapidFuzz/commit/5074585b9e97039544d1d020a48b174284926dfe.